helm-charts/splunk-otel-collector/templates/pdb-gateway.yaml

```diff
@@ -1,5 +1,6 @@
 {{ $gateway := fromYaml (include "splunk-otel-collector.gateway" .) }}
-{{- if and $gateway.enabled .Values.podDisruptionBudget }}
+{{ $gatewayEnabled := eq (include "splunk-otel-collector.gatewayEnabled" .) "true" }}
+{{- if and $gatewayEnabled .Values.podDisruptionBudget }}
```

Comment:
This is a breaking change. `splunk-otel-collector.gatewayEnabled` should use `splunk-otel-collector.gatewayEnabled` helper instead of `gateway.enabled` to keep the same behavior